docs/configuration.md

```diff
@@ -28,6 +28,14 @@ Adds a [Google Site Verification code](https://support.google.com/webmasters/ans
 google_site_verification: TvDTuyvdstyusadrCSDrctyd
 ```
 
+## `search`
+
+Enables search functionality. This indexes pages only and is not recommended for single-page sites.
+
+```yaml
+search: true
```

Comment:
Perhaps this can be `enable_search` for consistency with other boolean options like `show_govuk_logo`?